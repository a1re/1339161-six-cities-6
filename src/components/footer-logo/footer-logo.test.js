import React from 'react';
import {render} from '@testing-library/react';
import FooterLogo from './footer-logo';

it(`Footer logo renders correctly`, () => {
  const {container} = render(<FooterLogo />);
  expect(container).toMatchSnapshot();
});
