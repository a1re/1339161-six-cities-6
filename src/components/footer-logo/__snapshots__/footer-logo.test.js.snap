// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer logo renders correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <a
      class="footer__logo-link"
      href="main.html"
    >
      <img
        alt="6 cities logo"
        class="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
